### PR TITLE
Added ion.rangeSlider

### PR DESCRIPTION
**Pull request by zippy1981**

### CODE REVIEW

File: files/ion.rangeslider/update.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "packageManager": "github",
+  "name": "ion.rangeSlider",
+  "repo": "IonDen/ion.rangeSlider",
+  "files": {
+    "include": [
+        "css/*.css",
+        "js/*.js",
```
Comment (jimaek) [inline files/ion.rangeslider/update.json]: Change it to *.min.js please to include only the minified files

**Comment by jimaek** [comment]
You also need to add `./` to the beginning of all of your paths to retain the directory structure.

**Comment by jimaek** [comment]
You need to add ./ to ALL your paths, not just css

**Comment by zippy1981** [comment]
Fixed!!!

File: files/ion.rangeslider/info.ini [hunk 2]
```diff
@@ -0,0 +1,5 @@
+author = "Denis Ineshin"
+github = "https://github.com/IonDen/ion.rangeSlider"
+homepage = "http://ionden.com/a/plugins/ion.rangeSlider/en.html"
+description = "cool, comfortable and easily customizable range slider"
+mainfile = "ion.rangeSlider.min.js"
```
Comment (jimaek) [inline files/ion.rangeslider/info.ini]: Correct would be `js/ion.rangeSlider.min.js`

**Comment by zippy1981** [comment]
I have fixed the main file

**Pull request merged**